app/os_windows.go

```diff
@@ -954,4 +994,64 @@ func configForDPI(dpi int) unit.Metric {
 	}
 }
 
+// schemesURI is a list of schemes, comma separated, that must be
+// defined using -X compiler ldflag, that used in gogio.
+var schemesURI string
+var schemesURIList []string
+
+// startupURI is the URI that started the app, if any.
+var startupURI string
```

Comment:
Do this need to be global? I don't see it used outside of the following `init`.

Reply:
It's used inside newWindow, and the URLEvent is sent if `startupURI` is not empty.